Cleanup some documentation, markdown formatting and broken links (#6890)
cri-o/cri-o: install.md

xUbuntu 18.04
```

> To install, choose a supported version for your operating system, and export it

Reviewer: Really "operating system" and not "CRI-O environment" or some such?

- `CentOS 8` (or higher): `pkgconfig` package is replaced by `pkgconf-pkg-config`
- By default btrfs is not enabled. To add the btrfs support, install the
  following package: `btrfs-progs-devel`

> - It is possible the distribution packaged version of runc is out of date.

Reviewer: ```suggestion
- The distribution packaged version of runc may need to be updated.
```

- `CentOS 8` (or higher): `pkgconfig` package is replaced by `pkgconf-pkg-config`
- By default btrfs is not enabled. To add the btrfs support, install the
  following package: `btrfs-progs-devel`
- It is possible the distribution packaged version of runc is out of date.
- If you'd like to get the latest and greatest runc, consider using the one

> - found in [devel:kubic:libcontainers:stable](https://build.opensuse.org/project/show/devel:kubic:libcontainers:stable)

Reviewer 2: can these be indented in one? ideally they'd be part of the same line, but if the linter is complaining we can separate

Author: It was quite hard to make the decision where to add line breaks consistently. I clearly have slipped up a couple of times due to the speed at which I was fixing the 1400 warnings.
The dash here is incorrect and I will include a bit more of the previous line in the next iteration.

**Caveats and Notes:**

If using an older release or a long-term support release, be careful to
double-check that the version of `runc` is new enough (running `runc --version`

> should produce `spec: 1.0.0`), or else build your own.

Reviewer: does 1.0.0 still hold?  Maybe add "or greater" type of thing.

If updating to a patch version (for example, ``VERSION=1.8.3``
  ), run

```shell
apt update cri-o cri-o-runc
```
<!-- markdownlint-enable MD024 -->

> Otherwise, be sure that the environment variable ```$OS``` is set to your operating

Reviewer: ```suggestion
Otherwise, be sure that the environment variable ```$OS``` is set to the appropriate value from the following table for your operating system.
```
